src/oop/Engineer.java

```diff
@@ -0,0 +1,19 @@
+package oop;
+
+public class Engineer extends Infomation {
+    public String major;
+    public void infoEngineer () {
+        this.major = "IT engineer";
+        this.age = 27;
+        this.name = " Nguyen Viet Vu";
+        this.address= "Ho Chi Minh City" ;
+
+    }
+    public void printInfo_Engineer () {
+        System.out.println("Major is:  " + major);
+        System.out.println("Age is:  " + age) ;
+        System.out.println("Name is:  " + name );
+        System.out.println("Address is:  " + address);
+
```

Comment:
In thông tin -> Nên có hàm tổng quát ở lớp cha
Lớp con kế thừa hàm và thêm thông tin
Từ khoá: Override, overload